src/guiding.jl

```diff
@@ -33,10 +41,10 @@ function (G::GuidingDriftCache)(du,u,p,t)
     # ν, P, c
     ν = @view soldis[1:d,cur_time]
     P = reshape(@view(soldis[d+1:d+d*d,cur_time]), d, d)
-  
+
     r = P\(ν - x)
 
-    du[end] = dot(f(x,p,t) -  b̃(x,k.plin,t), r) - 0.5*tr((outer_(g(x,p,t)) - σ̃(x,k.plin,t)*σ̃(x,k.plin,t)')*(inv(P) - r*r'))
+    du[end] = dot(f(x,p,t) - b̃(x,k.plin,t), r) - 0.5*tr((outer_(g(x,p,t)) - outer_(σ̃(x,k.plin[3],t)))*(inv(P) - r*r'))
```

Comment:
@mschauer I added the second `outer_` here.. is this correct? I thought this is needed if σ̃  evaluates to a vector.

Reply:
Looks good, I alway think of σ̃ as a matrix, so I missed this